### DISCUSSION TITLE
no GIS height data in edit mode

### DISCUSSION THREAD

**Discussion by Wolfgang45** [domlysz/BlenderGIS#611] [Q&A]
GIS EXPORT_GOOGLE_SAT_WM flattens when I switch to edit mode.
Why? 
How to fix this - showing height relief in blender edit mode?
I use blender 2.93 and GIS 2.2.5
Thank you.

**Comment by andrel-home** [comment]
I think you just need to apply all the modifiers.

**Reply by Wolfgang45** [reply]
Thanks,
that helped figure it out. I am very new to Blender
